src/Infrastructure.VS/Roslyn/SolutionRoslynAnalyzerManager.cs

```diff
@@ -72,8 +73,15 @@ public SolutionRoslynAnalyzerManager(
         connectedModeAnalyzerProvider.AnalyzerUpdatedForConnection += HandleConnectedModeAnalyzerUpdate;
     }
 
-    public void OnSolutionChanged(string solutionName, BindingConfiguration bindingConfiguration)
+    public async Task OnSolutionBindingChangedAsync(string solutionName, BindingConfiguration bindingConfiguration)
```

Comment:
Is binding configuration still needed here?

Reply:
Doesn't seem so, no